### PR TITLE
Fix export

### PR DESCRIPTION
**Pull request by CedNaru**
Two issues were solved :
- I forgot to fill the map that link source files to names in release mode.
- Source files were exported with a different extension and not picked up by our custom loader.

I tested:
- Windows Editor (with reloading)
- Windows Export
- Android Export

### CODE REVIEW

File: src/kotlin_editor_export_plugin.cpp [hunk 1]
```diff
@@ -146,35 +146,10 @@ String KotlinEditorExportPlugin::get_name() const {
...
-        exported_script.instantiate();
-        return exported_script;
-    } else if (ext == GODOT_JAVA_SCRIPT_EXTENSION) {
-        Ref<JavaScript> exported_script;
-        exported_script.instantiate();
-        return exported_script;
-    } else {
-        return {};
+    if (ext == GODOT_JVM_REGISTRATION_FILE_EXTENSION || ext == GODOT_KOTLIN_SCRIPT_EXTENSION || ext == GODOT_JAVA_SCRIPT_EXTENSION) {
+        add_file(p_path, Vector<uint8_t>(), true);
```
Comment (chippmann) [inline src/kotlin_editor_export_plugin.cpp:152]: I would add a comment on what we do here and why

**Review by chippmann — CHANGES_REQUESTED** [review]
Will test exports and editor on linux later today

**Review by chippmann — APPROVED** [review]
Editor and exports work on Linux

**Review by piiertho — APPROVED** [review]
Tested on iOS

**Pull request merged**